playbook_launch_web_instance.yml

```diff
@@ -2,7 +2,7 @@
 - name: Setup Docker Server
   hosts: local
   connection: local
-  gather_facts: False
+#  gather_facts: False
```

Comment:
Might want to revert this sometimes gathering facts causes issues when python is not installed.